server/xpub-model/entities/file/index.test.js

```diff
@@ -11,6 +11,31 @@ describe('File', () => {
     await createTables(true)
   })
 
+  describe('constructor', () => {
+    it('sets the status to undefined, even with a default within the schema', async () => {
+      const file = new File({
+        manuscriptId: '1',
+        filename: 'thisfile.txt',
+        url: '/an/url',
+      })
+      expect(file.status).toBe(undefined)
+    })
+  })
+
+  describe('save()', () => {
+    it('should set the status to CREATED', async () => {
+      const manuscript = await new Manuscript({
+        createdBy: userId,
+      }).save()
+      const file = await new File({
+        manuscriptId: manuscript.id,
+        filename: 'thisfile.txt',
+        url: '/an/url',
+      }).save()
+      expect(file.status).toBe('CREATED')
+    })
+  })
```

Comment:
I added these tests to make it clear that the status of a file only gets set when `.save()` is called, even if we have a default value within the model schema.

This doesn't affect `manuscript.fileStatus` since `manuscript.files` is setup via a relation and hence only gets populated from the database.